data/dev/dataset.csv

```diff
@@ -9,4 +9,5 @@ The assembled scaffolds of high quality sequences total 1.23 Gb, with the contig
 668,38,669,200070,supporting data for nothing in particular,"well now, how to describe nothing in particular?",32,https://ftp.cngb.org/pub/gigadb/pub,Private,,,,,1,ImP3Bbu7ytRSfYFh,,38,none,
-700,14,700,300070,test dataset,"test generic image will be display for no image dataset",32,https://ftp.cngb.org/pub/gigadb/pub,Published,,,,,1,,,38,none,
\ No newline at end of file
+700,14,700,300070,test dataset,"test generic image will be display for no image dataset",32,https://ftp.cngb.org/pub/gigadb/pub,Published,,,,,1,,,38,none,
+5,345,5,100039,Genomic data of the Puerto Rican Parrot (<em>Amazona vittata</em>) from a locally funded project. ,"These data represent the first assembly of a genome sequence for a critically endangered parrot (<em>Amazona vittata</em>) endemic to the United States, and also the first genome of a species from the diverse and ecologically important genus <em>Amazona</em> native to South America and the Caribbean. One sample has been selected from the non-reproductive female at Rio Abajo Breeding Facility in Puerto Rico (IACUC#201109.1), and sequenced on Illumina HiSeq platform with both fragment and paired-end sequencing approaches, resulting in a total of 42,479,499,706 bases. We predicted a total coverage depth of 26.89X of the parrot’s genome: 17.08X coverage for the short fragment reads, and 9.8X coverage for the mate pairs. The sequencing was initiated with the construction of two genome libraries: a short fragment library (~300 bp inserts) for sequencing the majority of the genome, and a long fragment library (~2.5 Kb inserts) to generate scaffolds to be used to order and assemble contigs derived from the short fragment library. The Illumina paired-end and mate-pairs reads were assembled together with Ray (<a href=""http://denovoassembler.sourceforge.net"">http://denovoassembler.sourceforge.net</a>), with the k-mer defined iteratively. In total, given that the genome size is predicted to be 1.58Gb, with the total scaffold length of 1,184, 594,388 bp, the overall coverage of the genome is around 76%, a value that might be slightly overestimated given that some of the scaffolds may be overlapping but could not be assembled. Filtering followed by assembly resulted in 259,423 contigs (N50=6,983 bp, longest = 75,003 bp), which was further combined into 148,255 scaffolds (N50 = 19,470, longest = 206,462 bp). The database contains all of the contigs, scaffolds, corresponding assembly parameters, and the annotations for the known repeats and coding sequences. The assembled scaffolds allow basic genomic annotation and comparative analyses with other available avian whole-genome sequences.",4294967296,ftp://climb.genomics.cn/pub/10.5524/100001_101000/100039/,Incomplete,GigaDBSubmissionForm_Parrot.xls,1586ed77abdec5b9f839be0f5d61f357,2012-09-11,2012-09-14,1,,,,,
```

Comment:
I think this record should be moved to the top so the records are in order of their ids.